### ISSUE TITLE
Issue with "HTMLCanvasElement.getContext()": (short summary here please)

### ISSUE THREAD

**Issue by PieNam** [mdn/content#5369]
MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/API/HTMLCanvasElement/getContext

#### What information was incorrect, unhelpful, or incomplete?

In the Chinese version of the document, about `contextAttributes` of `WebGL`.

'premultipliedAlpha: 表明排版引擎讲假设绘制缓冲区包含预混合alpha通道的boolean值。'

should be:

'premultipliedAlpha: 表明排版引擎【将】假设绘制缓冲区包含预混合alpha通道的boolean值。'

#### Specific section or headline?

语法 - 参数 - 上下文属性 - WebGL 上下文属性

#### What did you expect to see?

`premultipliedAlpha: 表明排版引擎将假设绘制缓冲区包含预混合alpha通道的boolean值。`

#### Did you test this? If so, how?

just a Chinese typo

<details>
<summary>MDN Content page report details</summary>

* Folder: `zh-cn/web/api/htmlcanvaselement/getcontext`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/API/HTMLCanvasElement/getContext
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/web/api/htmlcanvaselement/getcontext/index.html
* Last commit: https://github.com/mdn/translated-content/commit/079fefb94cacbc3d34fc5fb4e274877a4339116c
* Document last modified: 2021-05-25T19:51:09.000Z

</details>

**Comment by teoli2003** [comment]
Maybe someone at @yari-content-zh can help here.

**Comment by sideshowbarker** [comment]
> Maybe someone at @yari-content-zh can help here.

Yeah I guess for these kinds of issues, we should start comment-mentioning the appropriate teams.

But that said, we can’t actually fix any of these issues with changes to https://github.com/mdn/content sources — instead, as far as I understand, the changes need to be made in the https://github.com/mdn/translated-content/ sources.

If so, then it seems like the best way to be triaging these issues is to be transferring them to the https://github.com/mdn/translated-content/ repo — using the **Transfer issue** button here.

But I personally can’t don’t that, because I don’t actually have perms to transfer any issues to https://github.com/mdn/translated-content/.

Not that it’s a huge number of issues — https://github.com/mdn/content/labels/l10n-zh shows only 5 open issues for the **l10n-zh** label. But still it seems like it would be better if they were getting tracked in the same repo where pull requests with fixes for them could actually be merged.

**Comment by teoli2003** [comment]
Absolutely, these issues should be transfered over (l10n-fr does this on a regular basis). Hence the ping.
